example/src/api/getDataSelection.ts

```diff
@@ -0,0 +1,33 @@
+import { LatLngTuple } from 'leaflet'
+
+const generateParams = (data: object) =>
+  Object.entries(data)
+    .filter(([, value]) => value !== null && value !== undefined)
+    .map((pair) => pair.map(encodeURIComponent).join('='))
+    .join('&')
+
+export type DataSelection = {
+  numberOfRecords: number
+  data: object[]
+  markers: LatLngTuple[]
+}
+
+export default async function getDataSelection(
```

Comment:
Missing await?

Reply:
using await here is redundant, as we return the promise immediately